src/chains/ethereum/ethereum/tests/api/evm/evm.test.ts

```diff
@@ -78,91 +79,139 @@ describe("api", () => {
       });
     });
 
-    describe("evm_mine", () => {
-      it("should mine `n` blocks on demand", async () => {
+    describe.only("evm_mine", () => {
```

Comment:
This is the real change to the tests. I'm running all of the pre-existing tests with both the "eager" and "strict" mode